week-2/interpret/invert.js

```diff
@@ -10,18 +10,33 @@ function invert(obj) {
   const invertedObj = {};
 
   for (const [key, value] of Object.entries(obj)) {
-    invertedObj.key = value;
+    invertedObj[value] = key;
   }
 
   return invertedObj;
 }
 
+console.log(invert({ x: 10, y: 20 }));
+
 // a) What is the current return value when invert is called with { a : 1 }
 
+// { a : 1 }
+
 // b) What is the current return value when invert is called with { a: 1, b: 2 }
 
+// { a: 1, b: 2 }
+
 // c) What is the target return value when invert is called with {a : 1, b: 2}
 
+// { 1 : a, 2 : b }
+
 // c) What does Object.entries return? Why is it needed in this program?
 
+// Object.entries creates an array of the key-value pairs. This allows us to identify in the for loop
+// what key and value should be paired. Then we can reverse the pair instead of the complete object?
```

Comment:
Just like you said, `Object.entries()` gives us an array of arrays, where the arrays are key object pairs. About the second part, sure you can use it to reverse the object, but you can also directly use the object while reversing it, just like you did on the example above.